assets/assets_identifiers_ws.py

```diff
@@ -17,3 +18,17 @@ def url(self):
 	def on_message(self,ws, message):
 		self.on_message_callback(StreamedIdentifiers.from_json(message))
 
+	def on_error(self,ws,error):
+		if type(error) == OSError:
```

Comment:
can you please add some comments explaining in what / why this error would happen?